src/applications/simple-forms/21-10210/pages/statementInformation2.js

```diff
@@ -0,0 +1,54 @@
+export default {
+  uiSchema: {
+    claimantType: {
+      'ui:widget': 'radio',
+      'ui:options': {
+        hideIf: formData => formData.claimOwnership === undefined,
+        updateSchema: (formData, schema, uiSchema) => {
```

Comment:
@ndsprinkle , this updataSchema call has disable-eslint code-comments which auto-include VA Platform CoP Frontend as PR-reviewer.  I need them for my uiSchema logic.
Should we try to get this much-smaller PR, though WIP, reviewed & approved now, instead of waiting to open a huge PR when everything's in?
Please let me know.  Thanks much!